src/services/domainService.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable prefer-destructuring */
 /* eslint-disable no-restricted-syntax */
 const config = require('config');
+// const crypto = require('node:crypto');
```

Comment:
this is being commented, can be deleted instead

Reply:
Removed